### ISSUE TITLE
Server/Configs/*.conf not updating

### ISSUE THREAD

**Issue by tiamat2012** [crowbarmaster/BedrockManagementService#14]
When I make changes in the client under Edit Server Config, the server.properties file changes, but the Server/Configs/<servername>.conf file does not.

The reason I even noticed this is because the Default world is set to creative/easy.  And I tried to change it via the client to survival and the world would start and the log would reflect the change, but my ipad client always put me in creative and I could not figure out why.  I was looking at all the config files trying to figure it out.  I guess that's how minecraft works and whatever game mode you joined the world on a server is set and you have to be an op to switch it.  I just wasn't expecting it.

Anyway, I am not sure what the .conf file is used for if it doesn't stay in sync with the server.properties file anymore.  Perhaps it's not needed anymore or this is just a bug.

**Comment by crowbarmaster** [comment]
Most definitely a bug we have here. These should always stay in sync, with the server.properties file following the conf file. The current flow is that a fresh copy of the server.properties file is written before the server process starts, so for these to be out of sync means that the conf was rewritten with the old one somehow. I will spend some good time checking the flow here. I consider this to be one of the more important attributes to this entire system.

**Comment by crowbarmaster** [comment]
Bug was located. Missing call to save file method added, and functions as it should again. Will be shipped in next release.

**Comment by tiamat2012** [comment]
Hmm - does that mean I will have to manually update the .conf file before I install the new update, otherwise it will deploy the out of sync contents of the conf file over the working server.properties?  If anyone else is using this software, they need to know what to do before installing the next version (yeah, I know, It may just be you and me, but I can't see the downloads stats to be certain).

What if I had a world already and wanted to use this bedrock service to run it?  Perhaps some instructions would be helpful to others.  What I did was create a new world in the client with all the correct settings that matched the world being imported, let it start up once, then I stopped the service, replaced the world subfolder files with the world being imported (but not the server.properties file which lives outside that folder), and then restarted the server.  Then I imported the packs so that the world*packs.json files would get created properly.  Perhaps a short migration guide for people would be useful for new adopters.
